src/web/webutils.cpp

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) rAthena Dev Teams - Licensed under GNU GPL
+// For more information, see LICENCE in the main folder
+
+
+#include "webutils.hpp"
+#include <string>
+#include <algorithm>
+#include <iostream>
+
+/* 
+ * Appends to_append to the JSON object obj
+ * modifies the obj parameter
+ * Returns true on succes, false on failure
+ * 
+ * Note: very hacky, if we ever add a json parser/emitter, replace this
+ * I just didn't want to add one just for this functionality
+ */
+bool addToJsonObject(std::string& obj, const std::string& to_append) {
```

Comment:
Should we really access std::cout directly here?
Usually we use our ShowMsg functions to make sure the logging level configured by the user is respected and to me those messages look like debug messages?